src/branchandbound.jl

```diff
@@ -0,0 +1,80 @@
+using TaylorModels
```

Comment:
it is not needed to do `using TaylorModels` as it has been already loaded in `taylormodels.jl`

Reply:
done